text/0000-channel-participation-api-without-system-channel.md

```diff
@@ -0,0 +1,497 @@
+channel participation API, and continue to operate the network in the usual way - creating channels through the system 
+channel. "Mixed mode" management, that is, using the system channel to create channels on some OSNs and the 
+participation API on other OSNs is not supported and highly discouraged.  
+
+## Channel participation commands 
+In this section channel participation commands are described in a high level. The commands are presented in REST style
+but some details (i.e. exact formats, responses) are omitted for clarity.
+   
+### Joining a channel
+A channel is joined by providing the OSN with a config block:
+
+  ```Join channel-name channel-config-block```
+  
+  or
+  
+  `POST /channels/channel-name` with body containing `channel-config-block`
+  
+The body contains a config block that contains all the details about the channel. 
+
+If the OSN is joining a new channel, the config block is the genesis block of the channel, generated by the 
+`configtxgen` tool, as described below. It looks like the first block of an 
+application channel that is generated by the OSN, when a channel creation transaction is submitted to the system channel.
+
+If the OSN is joining an existing channel, the config block is simply the last config block of the channel, retrieved 
+from one of the other OSNs.
+
+If the channel already exists, the call will fail.
+
+If the system channel exists, the call will fail. 
+          
+The OSN receiving this call will inspect the block and conduct some sanity checks in order to make sure that the block 
+is well-formed.
+
+The OSN will try to find itself in the consenters set, and determine if it is a follower or a member of the cluster
+executing consensus. 
+
+If it is not in the set, it will continue as a **follower**, and start to pull blocks from other OSNs. It will continue
+to do so until it catches up with the config block received in the invocation. From then on, it will continue to pull blocks, 
+but will inspect each incoming config block and check whether it is included in the consenters set. If it is, it will 
+start the runtime components that execute consensus, thus joining the cluster. 
+
+If the OSN is in the consenters set, it will continue as **member**, first pulling blocks from other OSMs util it catches
+up with the given config block, and then starting the runtime components that execute consensus, thus joining the cluster.
+
+Note that joining an OSN as a member means that the channel was updated to include that OSN in the consenters set before it
+was joined by the participation API. Therefore, during the catch-up process and until the new OSN starts executing 
+consensus, the cluster is a state of reduced fault tolerance. For long chains this might be a problem, since catch-up
+can take a long time. The solution is to join the new OSN as a follower. After it catches up with the cluster the 
+channel admin can update the channel config and include it in the consenters set.
```

Comment:
```
The solution is to join the new OSN as a follower. After it catches up with the cluster the 
channel admin can update the channel config and include it in the consenters set.
```
This is actually a new feature as well, correct?

Reply:
Yes, this is new feature.